Microsoft.DotNet.Interactive.Tests/LanguageKernelFormattingTests.cs

```diff
@@ -274,27 +274,5 @@ void f()
                 .And
                 .Contain("the-outer-exception");
         }
-
-        [Theory()]
-        [InlineData("Write-Warning 'I am a warning message'", "<pre>WARNING: I am a warning message</pre>")]
-        [InlineData("Write-Verbose 'I am a verbose message' -Verbose", "<pre>VERBOSE: I am a verbose message</pre>")]
-        [InlineData("'I am output'", "<pre>I am output</pre>")]
-        [InlineData("Write-Debug 'I am a debug message' -Debug", "<pre>DEBUG: I am a debug message</pre>")]
-        [InlineData("Write-Host 'I am an information message'", "<pre>I am an information message</pre>")]
-        [InlineData("Write-Error 'I am a non-terminating error'", "<pre>Write-Error: I am a non-terminating error</pre>")]
-        public async Task powershell_stream_formatting(string code, string output)
-        {
-            var kernel = CreateKernel(Language.PowerShell);
-            var command = new SubmitCode(code);
-            await kernel.SendAsync(command);
-
-            KernelEvents
-                .Should()
-                .ContainSingle<DisplayedValueProduced>()
```

Comment:
Are these events no longer produced?

Reply:
This test is removed because the related code `StreamHandler.cs` is now removed.
previously the PS kernel didn't have a host, so handlers are registered to all the streams that poweshell produces which publish `DisplayedValueProduced` events for all produced records.

Now with the host implementation, `StandardOutputValueProduced` messages are produced by the host directly calling `Console.WriteLine` and `Console.Write`.
The only `DisplayValueProduced` message in use is for rendering the progress bar in powershell. That's the method you asked to create a non-aysnc version.